src/markdown-renderer.ts

```diff
@@ -99,16 +98,18 @@ export default class MarkdownRenderer {
     if (issue) {
       let markdown = "";
 
-      if (issue.number && issue.pull_request && issue.pull_request.html_url) {
-        const prUrl = issue.pull_request.html_url;
+      if (issue && issue.html_url) {
+        const prUrl = issue.html_url;
         markdown += `[#${issue.number}](${prUrl}) `;
       }
 
       if (issue.title && issue.title.match(COMMIT_FIX_REGEX)) {
         issue.title = issue.title.replace(COMMIT_FIX_REGEX, `Closes [#$3](${this.options.baseIssueUrl}$3)`);
       }
 
-      markdown += `${issue.title} ([@${issue.user.login}](${issue.user.html_url}))`;
+      if (issue.user) {
```

Comment:
again this is new behaviour. If we throw an error for an unknown user we will never have a situation where a user is not available right?

Reply:
it was because if TS types. I changed it to !!